core-primitives/extrinsics-factory/src/lib.rs

```diff
@@ -172,7 +179,10 @@ pub mod tests {
 			node_metadata_repo,
 		);
 
-		let opaque_calls = [OpaqueCall(vec![3u8; 42]), OpaqueCall(vec![12u8, 78])];
+		let opaque_calls = [
+			(OpaqueCall(vec![3u8; 42]), GenericMortality::immortal()),
+			(OpaqueCall(vec![12u8, 78]), GenericMortality::immortal()),
```

Comment:
Oi, can we document those what is in those `vec!`?

Reply:
dummy values for unit test. don't care